lib/Optimizer/Transforms/GenDeviceCodeLoader.cpp

```diff
@@ -105,8 +105,8 @@ class GenerateDeviceCodeLoader
           LLVM_DEBUG(llvm::dbgs() << "adding declaration: " << op);
           declarations.push_back(&op);
         }
-      } else if (auto ccGlobalOp = dyn_cast<cudaq::cc::GlobalOp>(op)) {
-        LLVM_DEBUG(llvm::dbgs() << "adding global constants: " << op);
+      } else if (auto globalOp = dyn_cast<cudaq::cc::GlobalOp>(op)) {
+        LLVM_DEBUG(llvm::dbgs() << "adding global: " << op);
```

Comment:
Is this not a constant any longer? That's a departure, since we shouldn't be creating global variables AFAIK.

Reply:
It is a constant still. I believe this is just a merge of the same change Pradnya and I had to do, so just a name change

Reply:
updated to match the current version in main